themes/opendk/index.js

```diff
@@ -1,3 +1,11 @@
 module.exports = function (app) {
-  // noop
-}
+  const dms = app.get('dms')
+  const config = app.get('config')
+  const Model = new dms.DmsModel(config)
+
+  app.get('/', async (req, res,next) => {
+      const collections = await Model.getCollections()
+      res.locals.collections = collections
+      next()
```

Comment:
Awesome 🌠

Reply:
@sagargg I've found cool solution on StackOverflow about how to get random `N` elements from an array with the length of `M`:

```
// Shuffle array
const shuffled = array.sort(() => 0.5 - Math.random());

// Get sub-array of first n elements after shuffled
let selected = shuffled.slice(0, n);
```

Note that `M` shouldn't be less than `N`.